test/util/bspNet/block.ts

```diff
@@ -25,6 +25,39 @@ export interface SealedBlock {
   extSuccess?: boolean;
 }
 
+export const extendFork = async (
+  api: ApiPromise,
+  options: { parentBlockHash: string; amountToExtend: number; verbose: boolean }
```

Comment:
How about making the verbose patameter optional?

```suggestion
  options: { parentBlockHash: string; amountToExtend: number; verbose?: boolean }
```